helpdesk_mgmt/models/helpdesk_ticket_team.py

```diff
@@ -61,4 +70,15 @@ def _compute_todo_tickets(self):
                     lambda ticket: ticket.unattended))
             record.todo_ticket_count_high_priority = len(
                 record.todo_ticket_ids.filtered(
-                    lambda ticket: ticket.priority == '3'))
+                    lambda ticket: ticket.priority == "3")
+            )
+
+    def get_alias_model_name(self, vals):
+        return 'helpdesk.ticket'
+
+    def get_alias_values(self):
+        values = super(HelpdeskTeam, self).get_alias_values()
+        values['alias_defaults'] = defaults = safe_eval(
```

Comment:
Why do yo need to do this manually?

Reply:
"Return values to create an alias, or to write on the alias after its creation"... I think that it is the better option to get alias values when you create or write from the helpdesk team link model

Reply:
But you don't have to parse existing default values... Just add what you want, but let super to do the rest. Or better, which is the core strategy, save proper defaults on alias creation.

Reply:
For the moment I take it off so I can do the merge, I would like to check the flow on sale team alias, and view if it makes sense in this case... 
https://github.com/OCA/OCB/blob/12.0/addons/crm/models/crm_team.py#L81
anyway we can see it in its pr.